packages/web3-providers/src/Web3/EVM/apis/ConnectionReadonlyAPI.ts

```diff
@@ -519,17 +519,26 @@ export class EVMConnectionReadonlyAPI
 
         const options = this.ConnectionOptions.fill(initial)
         const NATIVE_TOKEN_ADDRESS = getTokenConstant(options.chainId, 'NATIVE_TOKEN_ADDRESS')
-        const BALANCE_CHECKER_ADDRESS = getEthereumConstant(options.chainId, 'BALANCE_CHECKER_ADDRESS')
         const entities: Array<[string, string]> = []
 
         if (listOfAddress.some(isNativeTokenAddress)) {
             entities.push([NATIVE_TOKEN_ADDRESS ?? '', await this.getBalance(options.account, options)])
         }
+        const BALANCE_CHECKER_ADDRESS = getEthereumConstant(options.chainId, 'BALANCE_CHECKER_ADDRESS')
+        if (!BALANCE_CHECKER_ADDRESS) {
+            if (process.env.NODE_ENV === 'development') {
```

Comment:
We could build it in those constant helpers

Reply:
It depends. Not all constant variables need to be checked.

Reply:
Got it. We can impl this if we need in the future.